### DISCUSSION TITLE
Autologon

### DISCUSSION THREAD

**Discussion by MarRich1** [TandoorRecipes/recipes#1329] [Q&A]
Hi,

I tried to find a docu how a guest login will work on local install tandoor docker installation but i didńt find anything
Can you point me to the right location or tell me how it can be achieved?

I would like to run a pi with a screen in the kitchen, and I don`t wont to logon there

**Comment by vabene1111** [comment]
You can invite a user via the space admin (top right drop down menu). That user can than be used on the screen.

**Reply by MarRich1** [reply]
But there is no possibility to login via url parameter, our as guest without any login or something like that?

**Reply by vabene1111** [reply]
no there currently isn't.
